volatility/framework/automagic/linux.py

```diff
@@ -310,13 +310,15 @@ def generate_kernel_handler_info(
             cls,
             context: interfaces.context.ContextInterface,
             layer_name: str,
-            kernel,  # ikelos - how to type this??
+            kernel_name,  # ikelos - how to type this??
```

Comment:
Looks good, this can be a `str` now presumably?

Reply:
good catch, missed that